pkgs/development/libraries/libsidplayfp/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, autoconf, automake, pkgconfig, libtool
+, docSupport ? true, doxygen ? null, graphviz ? null }:
+
+assert docSupport -> doxygen != null && graphviz != null;
+
+stdenv.mkDerivation rec {
+  pname = "libsidplayfp";
+  major = "1";
+  minor = "8";
+  level = "6";
+  version = "${major}.${minor}.${level}";
+  name = "${pname}-${major}.${minor}.${level}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/sidplay-residfp/${pname}/${major}.${minor}/${pname}-${major}.${minor}.${level}.tar.gz";
+    sha256 = "0lzivfdq0crmfr01c6f5h883yr7wvagq198xkk3srdmvshhxmwnw";
+  };
+
+  buildInputs = [ autoconf automake libtool pkgconfig ]
```

Comment:
Why are the autotools dependencies needed? There is no call to `autoreconf` or `autogen.sh` and the lib still compiles after removing `autoconf automake libtool`. Also, these should be `nativeBuildInputs`.